build.sbt

```diff
@@ -112,12 +112,13 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core" % "2.3.1",
       "org.typelevel" %%% "cats-laws" % "2.3.1" % Test,
-      "org.typelevel" %%% "cats-effect" % "3.0.0-M5",
-      "org.typelevel" %%% "cats-effect-laws" % "3.0.0-M5" % Test,
-      "org.typelevel" %%% "cats-effect-testkit" % "3.0.0-M5" % Test,
+      "org.typelevel" %%% "cats-effect" % "3.0-65-7c98c86",
```

Comment:
We need this version to pick up (1) adjustment to MonadCancel laws (2) fix in TestInstances for an integer overflow